src/Tools/BuildUtil/BuildUtil/WriteCodeFragmentEx.cs

```diff
@@ -1,186 +0,0 @@
-﻿using Microsoft.Build.Framework;
```

Comment:
Correct. It was being used before MSBuild 15.0 to work around a bug in WriteCodeFragment. I forgot to delete this code once we moved back to the standard task.